docs/readme-nodes-configuration.md

```
@@ -389,26 +391,24 @@ The following section explains Portal-specific parameters related to entity type
 A title resolver configuration allows to fetch micro frontend specific data outside the micro frontend application. This data can then be used to show a dynamic navigation header.
 
 - **titleResolver** - basically a [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) configuration including a simple result processing to extract display title and image URL data. The configuration object has the following properties:
-    - **request** - the request configuration consists of a **url** property to fetch the data from, as well as optional **method**, **headers** and **body** properties (see corresponding parameters [here](https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters)).
+  - **request** - the request configuration consists of a **url** property to fetch the data from, as well as optional **method**, **headers** and **body** properties (see corresponding parameters [here](https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters)).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)